packages/regl-worldview/src/commands/GLTFScene.js

```diff
@@ -116,13 +116,12 @@ const drawModel = (regl) => {
   // build the draw calls needed to draw the model. This will happen whenever the model changes.
   const getDrawCalls = memoizeWeak((model: GLBModel) => {
     // upload textures to the GPU
-    const { images, accessors } = model;
+    const { accessors } = model;
     const textures =
       model.json.textures &&
-      images &&
       model.json.textures.map((textureInfo) => {
         const sampler = model.json.samplers[textureInfo.sampler];
-        const bitmap: ImageBitmap = images[textureInfo.source];
+        const bitmap = model.images && model.images[textureInfo.source];
```

Comment:
Is this just a revert or why is it changing? (I don't remember whether `images` always exists; I think it does but might be empty?)

Reply:
this is kind of a revert - I just added the `model.images &&` to ensure that there wasn't a flow error. `images` does not always exist.